src/Installation/wine.md

```diff
@@ -0,0 +1,18 @@
+# Installing wine
+
+A wine version higher than 7 is required, 8.3+ is recommended.
+You can see your wine version with `wine --version`
```

Comment:
This may not work on flatpak, `vinegar exec --version`

Reply:
Thanks for the input, im adding everything right now